[2022-12-04T16:12:01.737+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T00:00:00+00:00 [queued]>
[2022-12-04T16:12:01.745+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T00:00:00+00:00 [queued]>
[2022-12-04T16:12:01.746+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:01.746+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:12:01.748+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:01.757+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-03 00:00:00+00:00
[2022-12-04T16:12:01.764+0000] {standard_task_runner.py:55} INFO - Started process 163 to run task
[2022-12-04T16:12:01.768+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-03T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp8apswmsy']
[2022-12-04T16:12:01.771+0000] {standard_task_runner.py:83} INFO - Job 4: Subtask notification
[2022-12-04T16:12:01.836+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-03T00:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:12:01.910+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T00:00:00+00:00
[2022-12-04T16:12:01.913+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:12:01.916+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-03']
[2022-12-04T16:12:01.935+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:12:01.938+0000] {subprocess.py:93} INFO - received data at 2022-12-03
[2022-12-04T16:12:01.940+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:12:01.970+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221203T000000, start_date=20221204T161201, end_date=20221204T161201
[2022-12-04T16:12:02.022+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:12:02.042+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
